wavefront/server/apps/floware/floware/services/scheduled_job_service.py

```
@@ -393,144 +438,397 @@ async def _execute_email_dynamic_query_job(self, payload: dict, job_timezone: st
+                rls_filter_str = await self._rls_filter_for_user(user_id)
+                rows = await self._fetch_dynamic_query_rows(
+                    datasource_plugin,
+                    yaml_query,
+                    query_id,
+                    rls_filter_str,
+                    filter_expr,
+                    offset,
+                    limit,
+                    params,
+                )
+            except Exception as exc:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)